src/lib/trust_system/peer_trust.ml

```diff
@@ -245,6 +257,8 @@ let%test_module "peer_trust" =
             Trust_response.Trust_increase 0.2
 
       let to_log t = (string_of_sexp @@ sexp_of_t t, [])
+
+      let is_reason_for_heartbeat _ = false
```

Comment:
This basically means that heartbeat messages for the daemon to the libp2p helper are disabled, right?

Reply:
Not asking for any changes right now, but just some questions on the thinking here. In the long run, we completely replace the daemon's trust system module. In a more ideal architecture, do you think the trust system is still the right place to implement the heartbeat logic? It seems to me that it might make more sense to live in the networking layer where we do peer interactions.